templates/index.html

```diff
@@ -45,302 +31,234 @@
+            <p class="flex-auto">Implementation of scalable modern data infrastructure tailored to your organization's needs and budget</p>
+            <p class="mt-6">
+            </p>
+          </dd>
+        </div>
+        <div class="flex flex-col">
+          <dt class="flex items-center gap-x-3 text-base/7 font-semibold text-gray-900">
+            <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" strokeWidth={1.5} stroke="currentColor"  class="size-8 flex-none text-gray-dark">
+              <path strokeLinecap="round" strokeLinejoin="round" d="M7.5 14.25v2.25m3-4.5v4.5m3-6.75v6.75m3-9v9M6 20.25h12A2.25 2.25 0 0 0 20.25 18V6A2.25 2.25 0 0 0 18 3.75H6A2.25 2.25 0 0 0 3.75 6v12A2.25 2.25 0 0 0 6 20.25Z" />
+            </svg>
+            Analytics & Reporting
+          </dt>
+          <dd class="mt-4 flex flex-auto flex-col text-base/7 text-gray-900">
+            <p class="flex-auto">Source-of-truth metrics and actionable insights to drive strategic decision-making across your organization</p>
+            <p class="mt-6">
+            </p>
+          </dd>
+        </div>
+      </dl>
+    </div>
+  </div>
+</div>
+
+<!--FREE CONSULTATION SECTION-->
+<div class="bg-blue">
+  <div class="px-6 py-24 sm:px-6 sm:py-32 lg:px-8">
+    <div class="mx-auto max-w-4xl text-center">
+      <h2 class="text-balance text-4xl font-semibold tracking-tight text-white sm:text-5xl">Free Consultation</h2>
+      <p class="mx-auto mt-6 max-w-4xl text-pretty text-lg/8 text-white">Take our 5-minute data maturity questionnaire and we'll follow up with a custom-built consultation to inform your analytics roadmap. This puts you first in line for a free 1:1 consultation with our Data experts to discover ways to improve trust in your metrics, improve analyst efficiency and reduce infrastructure costs.</p>
+      <div class="mt-10 flex items-center justify-center gap-x-6">
+        <a href="/free-assessment" class="rounded-md bg-white px-3.5 py-2.5 text-sm font-semibold text-indigo-600 shadow-sm hover:bg-indigo-50 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-white">Get Free Recommendations</a>
       </div>
-  </section>
-  <!-- Team-->
-  <section class="page-section bg-light" id="team">
-      <div class="container">
-          <div class="text-center">
-              <h2 class="section-heading text-uppercase">Our Team</h2>
-              <h3 class="section-subheading text-muted">With specialties spanning data science, data engineering, and product analytics our team has experience on a wide range of analytical problems.</h3>
+    </div>
+  </div>
+</div>
+
+<!--BLOG SECTION-->
+<div class="bg-white py-24 sm:py-32">
+  <div class="mx-auto max-w-7xl px-6 lg:px-8">
+    <div class="mx-auto max-w-2xl text-center">
+      <h2 class="text-balance text-4xl font-semibold tracking-tight text-gray-900 sm:text-5xl">Recent Blog Posts</h2>
```

Comment:
```suggestion
      <h2 class="text-balance text-4xl font-semibold tracking-tight text-gray-900 sm:text-5xl">Our Featured Research</h2>
```
Rather than the latest, can we pin our 3 "best" articles to draw people in (e.g. Grid's 2mo payback article or Workforce Enhancement)?

Reply:
Yeah -- I made this so it gets any three articles that have the tag "data" and then gets the most recent three - so if you want to curate three we can. Just let me know which three you want!

Reply:
I like the intention, but I disagree here. It's pretty commonplace to just have your most recent blog posts previewed at the bottom of websites, so people come to expect that's what you're showing them if you preview 3 here. 

I wouldn't want to give off the impression to visitors that we rarely ever write blog posts if they see the same 3 every time they open the website, and some of the dates are from 1+ year ago